docs/_docs/1_4_powered_by.md

```diff
@@ -125,6 +125,7 @@ Using Hudi at Yotpo for several usages. Firstly, integrated Hudi as a writer in
 
 17. ["Next Generation Data lakes using Apache Hudi"](https://docs.google.com/presentation/d/1y-ryRwCdTbqQHGr_bn3lxM_B8L1L5nsZOIXlJsDl_wU/edit?usp=sharing) - By Balaji Varadarajan and Sivabalan Narayanan, Sep 2020, ["ApacheCon"](https://www.apachecon.com/)
 
+18. ["Landing practice of Apache Hudi in T3go"](https://drive.google.com/file/d/1ULVPkjynaw-07wsutLcZm-4rVXf8E8N8/view?usp=sharing) - By VinoYang and XianghuWang, November 2020, Qcon.
```

Comment:
put it on cn docs as well?

Reply:
+1 in general, we need to look at how they have diverged now.

Reply:
> put it on cn docs as well?

good idea, will add soon